EIPS/eip-6800.md

```diff
@@ -42,7 +42,7 @@ def group_to_field(point: Point) -> int:
     if point == bandersnatch.Z:
         return 0
     else:
-        return int.from_bytes(point.serialize(), 'little') % BANDERSNATCH_MODULUS
+        return point.map_to_scalar_field()
```

Comment:
This was not correct, since currently in our implementations the way we map an elliptic curve point to a scalar-field element is by doing `X/Y` (affine coordinates), and not serializing the point.

The change I propose is somewhat opaque on what exactly is `map_to_scalar_field()`. Another possibility is being more explicit about coordinates and do:
`return int.from_bytes(point.X/point.Y, 'little') % BANDERSNATCH_MODULUS`

But not sure about it.

If we rely on the current proposed change, whenever we define a cryptography-spec repository the reader can simply look at the code there to have a precise definition (and probably simplify the EIP here).